### PR TITLE
Jwmf new search bar

### PR DESCRIPTION
**Pull request by meredithkfwilliams**
- Added search functionality on vendor page
- Added feature tests for search functionality

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -55,12 +55,12 @@
...
-            <li><a href="#">Link 2</a></li>
-            <li><a href="#">Link 3</a></li>
-            <li><a href="#">Link 4</a></li>
-          </ul>
+
+          <div>
+              
+          </div>
+
+          </div>
```
Comment (cmkoller) [inline app/views/layouts/application.html.erb:63]: If this stuff is all empty, get rid of it!

File: spec/features/user_searches_for_vendor.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'visitor searches for a vendor', %{
+  As a visitor
+  I want to search for a vendor by name
+  So that I can find my favorite vendor
+} do
+
+  scenario 'visitor searches for a vendor by name' do
```
Comment (cmkoller) [inline spec/features/user_searches_for_vendor.rb:9]: Change this scenario description to be more descriptive of what you're actually testing here!

File: app/models/vendor.rb [hunk 3]
```diff
@@ -6,4 +6,12 @@ class Vendor < ActiveRecord::Base
...
 
   validates_presence_of :vendor_name
+
+  def self.approve(vendor)
+    vendor.update(viewable: true)
+  end
+
+  def self.search(query)
+    where("vendor_name ilike ?", "%#{query}%")
+  end
```
Comment (cmkoller) [inline app/models/vendor.rb:16]: There should be a unit test for this method!

**Comment by iEv0lv3** [comment]
We will update based on your comments in our new JWMF-carrierwave branch since this branch has been merged in!

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (iEv0lv3) [inline app/views/layouts/application.html.erb:63]: Updated! Pre-planning gone awry!

File: spec/features/user_searches_for_vendor.rb (hunk 2 above)
Comment (iEv0lv3) [inline spec/features/user_searches_for_vendor.rb:9]: Updated to be more descriptive!

**Comment by cmkoller** [comment]
Okay, that sounds good! So you know, you're able to keep working on a branch after it's been merged - if you just push a new commit on this branch, you'll have the option to open a new PR :) But doing the work in another branch is fine too!

**Pull request merged**